Setting/Factions/The Coven.md

```diff
@@ -0,0 +1,32 @@
+---
+tier: "5"
+allies:
+  - "[[Scrappers]]"
+enemies:
+  - "[[Ship Defense Corps]]"
+  - Intership Council
+demonym: cadmus
+aliases:
+  - CADMUS
+---
+#faction
+
+## Background
+They call themselves the Coalition for Biomorphological Equality, but to anyone else they are the Coven, a disorganized group aggressively bent on the democratization of biomodification and freedom to use morphogenic compounds. Those cogent enough often stage violent protests, of [[The Coven|CADMUS]] exclusive research. The [[Ship Defense Corps]] have been known to break up such altercations, by whatever means necessary.
```

Comment:
Why does Cadmus link to the coven page here?

Reply:
oops, I hadn't notice in obsidian the way this gets formatted